README.template.md

```diff
@@ -0,0 +1,47 @@
+{{ ossHeader }}
+
+## Getting Started :running:
+
+### Requirements
+
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+- At least PHP 7
+- The grpc PHP extension. See the [gRPC docs](https://github.com/grpc/grpc/blob/v1.46.3/src/php/README.md) section on installing the extension.
+
+**IDE Notes**: You'll most likely want to use an IDE that supports PHP development, such as [PhpStorm](https://www.jetbrains.com/phpstorm/) or [Microsoft Visual Studio Code](https://code.visualstudio.com/).
+
+### Installation
+
+Install composer [as described on the composer website](https://getcomposer.org/doc/00-intro.md).
+
+Add our repository to your `composer.json` file and our SDK as a dependency:
+
+```json
+{
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/momentohq/client-sdk-php"
+    }
+  ],
+  "require": {
+    "momentohq/client-sdk-php": "dev-main"
+  }
+}
+```
+
+Run `composer update` to install the necessary prerequisites.
+
+### Usage
+
+Check out full working code in [the examples directory](examples/) of this repository!
```

Comment:
Might be worth going ahead and slurping the example script into here?  Just to give the page a little more meat?

Not a blocker.  If you do wanna do it though it's just this:

https://github.com/momentohq/client-sdk-javascript/blame/main/README.template.md#L24

Reply:
Good idea, I added it, thanks.